app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -63,11 +62,11 @@ class ProfileFragment : Fragment() {
 
         profileFragmentViewModel.user.observe(this, Observer {
             it?.let {
-                rootView.name.text = "${it.firstName.nullToEmpty()} ${it.lastName.nullToEmpty()}"
+                val userName = "${it.firstName.nullToEmpty()} ${it.lastName.nullToEmpty()}"
+                rootView.name.text = userName
```

Comment:
Why this change?

Reply:
> Why this change?

It shows warning so I pass it as String. Should I recover it?

Reply:
What warning does it show? Ideally, we only fix issue related things in one PR